src/components/slideshow/slideshow.js

```diff
@@ -230,6 +255,13 @@ export default function (options) {
         dialog.addEventListener('close', onDialogClosed);
 
         loadSwiper(dialog, options);
+
+        const btnSlideshowExit = dialog.querySelector('.btnSlideshowExit');
+        if (btnSlideshowExit) btnSlideshowExit.classList.add('hide');
```

Comment:
Same problem as when calling `startHideTimer` - you cannot close slideshow on mobile. You can activate "device mode" in Chrome or Firefox using some phone/tablet profile to test it.
Let's do it for Desktop layout only.
Moreover, I think we can hide/show a top button group instead of a single `Exit` button.

```suggestion
        if (layoutManager.desktop) {
            const topActionButtons = dialog.querySelector('.topActionButtons');
            if (topActionButtons) topActionButtons.classList.add('hide');
        }

```